unit-tests/live/streaming/test-y16.cpp

```diff
@@ -0,0 +1,52 @@
+// License: Apache 2.0. See LICENSE file in root directory.
+// Copyright(c) 2022 Intel Corporation. All Rights Reserved.
+
+//#test:device D400*
```

Comment:
maybe the missing space after the # cause it not to run?

Reply:
It's not that I checked, windows + Linux did build and run this test.
Jetson didn't
@maloel  any reason why?

Reply:
Hmm looks like it wasn't even build on Jetson.
Not sure...
This is also before I made all the fixes to LibCI.
Maybe Remi can rebase and see what the new behavior is?

Reply:
@remibettan please update after rebase if this test runs on Jetson

Reply:
Problem was the Jenkins script that runs Jetson: it was always running on a `D4XX_MIPI` branch (on Remi's fork) which hasn't been updated in a while and obviously does not include his new changes, hence you cannot see the test running...

I fixed and submitted a PR in deploy. @Nir-Az waiting for your code review there...

Reply:
A new build is running:
https://rsjenkins.iil.intel.com/job/LRS_jetson_compile_pipeline/136/console

Reply:
Now it's working :) thanks